content/what-is-learnpack.md

```diff
@@ -0,0 +1,47 @@
+
+### Start LearnPack
+
+When you're in your program, you'll find practice exercises in your modules. When you open them, at the top of the screen, you'll see a blue box that looks like this:
+
+![learnpack](https://breathecode.herokuapp.com/v1/media/file/learnpack-1-png?raw=true)
+
+After you click the **Start New Exercise** button, another screen will pop up where you can set up your workspace. At the bottom of the screen, you'll see a green button that says **Create codespace** to make it.
+
+![Create Codespace](https://breathecode.herokuapp.com/v1/media/file/create-codespaces-png?raw=true)
+
+One great thing about learnPack is that it comes included with the code editor! You don't have to worry about starting it elsewhere. Once you open your codespace, LearnPack will start automatically, showing you the instructions on the right side of the screen.
+
+![Start LearnPack](https://breathecode.herokuapp.com/v1/media/file/learnpack-png?raw=true)
+
+> If for some reason LearnPack doesn't start, you just have to type in the terminal "learnpack start" 
+
+### LearnPack Functions
+
+LearnPack does more than just give you instructions during exercises. It's super useful in many ways! Right at the top of the instructions, there's a menu where you can find different things:
+
+![LearnPack Functions](https://breathecode.herokuapp.com/v1/media/file/learnpack-function-png?raw=true)
+
+- 📖 **Menu:** A menu will pop down if you click on the three bars in the top left corner. It has all the exercises sorted by sections, which helps you find them quickly.
+
+- 📦 **Build:** The "Build" button in programming is like putting your code together, so the computer can understand it. When you click "Build," the computer checks if your code is written correctly and turns it into something it can run. This needs to happen before you can see if your program works.
+
+- 🎓 **Tutorials:** When you're stuck, give it a shot first. Research and look closely at your code. If you still can't figure it out, we've got your back with video tutorials. Just click on the graduation cap icon, and it'll open the tutorial that will guide you through the exercise step by step.
+
+- ☑️ **Test:** After completing your code, click on the checkmark icon to test it. This ensures there are no errors, allowing you to move on to the next exercises!
+
+- 🗣️ **Language:** Where you see the flag icon, you can switch languages! We suggest choosing your native language or the one you're most comfortable with. This way, you'll understand the instructions better.
+
+- 🐞 **Report bugs:** Your feedback greatly benefits us! We're committed to making your learning experience the best it can be. If you notice something isn't working quite right or could be better, just click on the bug symbol. Our team will take a look and make the needed improvements. Your input helps us make things even better for everyone!
```

Comment:
~just click on the bug symbol~ just click on the bug symbol and describe the the problem you found